daemon.c

```diff
@@ -847,6 +847,11 @@ static SVCXPRT *create_tcp_transport(unsigned int port)
 	    fprintf(stderr, "Couldn't bind to tcp port %d\n", port);
 	    exit(1);
 	}
+	if (listen(sock, SOMAXCONN)) {
```

Comment:
Je mi trochu divné, že listen je potřeba jen v `create_tcp_transport()` a ne jinde. Pokud to testujete jen s TCP a UDP neřešíte, tak je potřeba to napsat do commit message.

Reply:
Protokol UDP nepoužívá listen. Testoval jsem i UDP a funguje.